website/blog/2020-08-04-prestodb-and-hudi.md

```diff
@@ -0,0 +1,153 @@
+
+This will be achieved by having mechanisms to refer to the external data files (from the source table) from within the new bootstrapped Hudi table. With the possibility of data residing in an external location (bootstrapped data) or under Hudi table’s basepath (recent data),   `FileSplit`s would require to store more metadata on these. This work would also leverage and build upon the Presto MOR query support we are adding currently.
+
+**[Support Incremental and point in time time-travel queries on Hudi tables](https://issues.apache.org/jira/browse/HUDI-887)**
+
+Incremental queries allow us to extract change logs from a source Hudi table. Point in time queries allows for getting the state of a hudi table between time T1 and T2. These are supported in Hive and Spark already. We are looking into supporting this feature in Presto as well. 
+
+In Hive incremental queries are supported by setting few configs in `JobConf` like for example - query mode to `INCREMENTAL`, starting commit time and max number of commits to consume. In Spark, there is a specific implementation to support incremental querying - `IncrementalRelation`. To support this in Presto, we need a way to identify the incremental query. Given Presto does not pass arbitrary session configs to the hadoop configuration object, an initial idea is to register the same table in the metastore as an incremental table. And then use query predicates to get other details such as starting commit time, max commits etc.
+
+**[RFC-15: Query planning and listing improvements](https://cwiki.apache.org/confluence/display/HUDI/RFC+-+15%3A+HUDI+File+Listing+and+Query+Planning+Improvements)**
+ 
+Hudi  write client and Hudi queries need to perform `listStatus` operation on the file system to get a current view of the file system. While at Uber, the HDFS infrastructure was [heavily optimized](https://eng.uber.com/scaling-hdfs/) for listing, this can be an expensive op for large datasets containing thousands of partitions and each partition having thousands of files on cloud/object storage. The above RFC work aims at eliminating list operation and providing better query performance and faster lookups, by simply incrementally compacting Hudi’s timeline metadata into a snapshot of a table’s state at that instant.
+
+The solutions here aim at 
+- ways for storing and maintaining metadata of the latest list of files 
```

Comment:
Capitalize the first letter of the next two sentences. Add a period at the end.
ways for -> Ways of